docker-compose.yml

```diff
@@ -29,5 +29,8 @@ rq_scheduler:
   links:
     - db
     - redis
+    # django-rq doesn't support rqscheduler `retry` mode yet
+    # so we need to use the original startup script, which retrieves
+    # the
   command:
-    python manage.py rqscheduler
+    rqscheduler --url=redis://redis:6379
```

Comment:
I wonder if we should simply set `RQ_REDIS_URL` as well and remove `--url ..`, to have that config value in one place instead of many? https://github.com/ui/rq-scheduler/blob/396efadda8610548b474e680507b278676fc2262/rq_scheduler/scripts/rqscheduler.py#L21